qemu/qemu_test.go

```diff
@@ -980,3 +981,22 @@ func TestBadCPUs(t *testing.T) {
 		t.Errorf("Error expected")
 	}
 }
+
+// -pci devices don't play well with Z hence replace them with corresponding -ccw devices
+// See https://wiki.qemu.org/Documentation/Platforms/S390X
+func init() {
+	if runtime.GOARCH == "s390x" {
+		deviceFSString = "-device virtio-9p-ccw,disable-modern=true,fsdev=workload9p,mount_tag=rootfs -fsdev local,id=workload9p,path=/var/lib/docker/devicemapper/mnt/e31ebda2,security_model=none"
```

Comment:
This entire block looks rather bogus to me - it hard-codes lots of values including docker paths, mac addresses, vsock cids, file names, file descriptors, network interfaces... and PCI details...?!

Reply:
@jodh-intel The other architecture use a similar string directly. So although, not optimal, hardcoding looks the way to go here.